blueprints/batch-jobs/README.md

```diff
@@ -0,0 +1,214 @@
+```
+The NGINX server and the 5-min job pods are rescheduled into the new c6g.2xlarge node, so **the job is restarted**, which is not ideal for batch jobs.
+
+After five minutes more, the job will finish, and Karpenter will replace the node with a **c6g.xlarge** instance (4 vCPU) for the NGINX server. You can repeat the previous steps to verify this behaviour.
+
+To clean up, execute:
+```
+kubectl delete -f workloads-evicted.yaml
+```
+To learn how to avoid this behaviour and wait for the job to be finished before replacing the node, go to [Preventing jobs of being evicted](#preventing-jobs-of-being-evicted).
+
+### Preventing jobs of being evicted
+If you executed the [optional](#optional-simulating-the-default-behaviour) part, make sure to delete the `workloads-evicted` deployment:
+```
+kubectl delete -f workloads-evicted.yaml
+```
+
+Let's start by deploying the workloads defined in the [workloads-not-evicted yaml](/karpenter-blueprints/blueprints/batch-jobs/workloads-not-evicted.yaml):
+```
+$> kubectl apply -f workloads-not-evicted.yaml
+deployment.apps/nginx created
+job.batch/2-min-job created
+job.batch/5-min-job created
+```
+This will create three pods that require **11 vCPU** in total:
+-  NGINX server - 2 vCPU required
+-  2-minutes job - 7 vCPU required
+- 5-minutes job - 2 vCPU required
+
+If you explore the [workloads-not-evicted yaml](/karpenter-blueprints/blueprints/batch-jobs/workloads-not-evicted.yaml), the `karpenter.sh/do-not-disrupt: "true"` annotations have been added to both jobs specifications.
+
+Go to [Results section](#results) to check the behaviour.
+
+***NOTE:***
+The sample deployment only allows scheduling pods on on-demand instances (`nodeSelector: karpenter.sh/capacity-type: on-demand`) to show the replace consolidation mechanism, as for spot nodes Karpenter only uses the deletion consolidation mechanism to avid breaking the price-capacity-optimized strategy, as explained [here](https://karpenter.sh/preview/concepts/disruption/#consolidation).
+
+## Results
+
+### Deployment verification
+Karpenter launches the cheapest EC2 instance for the workloads with at least **11 vCPU**: a **c6g.4xlarge** on-demand instance (16 vCPU, 32 GiB). You can check this by executing:
+```
+kubectl get nodes --label-columns node.kubernetes.io/instance-type
+```
+You should see something similar to this, where the new instance has been created a few seconds ago for the new workloads (labeled with `intent: apps`). Notice that there are two m4.large nodes created previously for the Karpenter pods (labeled with `intent: control-apps`):
```

Comment:
Let's remove this whole paragraph, or simply change it to highlight that there's a new node.